### ISSUE TITLE
[IMPROVEMENTS]: Incinerator Object

### ISSUE THREAD

**Issue by sourcedv** [typical-developers/oaklands-feedback#1125]
### Roblox Username

SOURCED_V

### What do you think can have quality-of-life improvements?

Incinerator Object

### How do you think things can be improved?

With someone that has a lot of loose items on their base, with an extreme laziness and inadequate financial capability to purchase storage crates to put said loose items on base, I'd like to have a way to easily delete loose items in bulk in circumstances where my storage system is too full.

**Comment by jadeneta** [comment]
Yes this is a necessity, with a filter like the filter conveyor has
